Apps/WebClient/src/ClientApp/src/components/report/ImmunizationHistoryReportComponent.vue

```diff
@@ -300,12 +306,12 @@ export default class ImmunizationHistoryReportComponent extends Vue {
             </b-table>
             <b-row class="mt-3">
```

Comment:
The `d-none d-md-block` classes could have been applied to this level, every element within is expected to behave the same way.

![image](https://user-images.githubusercontent.com/19548348/223255948-e8525d0e-3e2e-4ed1-b770-a06381bba0cb.png)